types/quill/index.d.ts

```diff
@@ -29,7 +30,11 @@ export type Sources = 'api' | 'user' | 'silent';
 
 export interface Key {
     key: string | number;
-    shortKey?: boolean;
+    shortKey?: boolean | null;
```

Comment:
whats the additional `null` for? people should really be passing a value or not setting it ideally

Reply:
I added null because, in the quill keyboard module, a modifier being set as null means that it can be active or not for the keyboard event to be fired. I.e., meaning that, for example, the shortKey can be true or false. For example:
```
// Only b with no modifier will trigger
quill.keyboard.addBinding({ key: 'B' }, handler);

// Only shift+b will trigger
quill.keyboard.addBinding({ key: 'B', shiftKey: true }, handler);

// Either b or shift+b will trigger
quill.keyboard.addBinding({ key: 'B', shiftKey: null }, handler);
```

Reply:
Very nice, totally wasn't aware of that usage. Good catch